recipe.py

```diff
@@ -0,0 +1,66 @@
+import re
+regex_name = "[a-zA-Z0-9- .]+$"
+class Recipe(object):
+    """
+      Users Class to handle methods to do with recipes categories and actual recipes
+      """
+    recipe_categories = {}
+    recipes = {}
+
+    def __init__(self, cat_name=None, owner=None, recipe_name=None):
+        """ Initializing  class instance variables"""
+        self.cat_name = cat_name
+        self.owner = owner
+        self.recipe_name = recipe_name
+
+    def category_register(self, cat_name, owner):
+        """ method that defines the elements required to create an account """
+        if re.match(regex_name, cat_name):
+            if cat_name != '' and cat_name.strip():
```

Comment:
You are checking if `cat_name` is not an empty string and not on if `cat_name` is `None`. Where is this handled?